Packs/VirusTotal/Integrations/VirusTotalV3/README.md

```diff
@@ -1,10 +1,13 @@
+# VirusTotal (API v3)
+
 This integration analyzes suspicious hashes, URLs, domains, and IP addresses.
 The integration was integrated and tested with version v3 API of VirusTotal.
+
 ## Configure VirusTotal (API v3) on Cortex XSOAR
 
 1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
-2. Search for VirusTotal (API v3).
-3. Click **Add instance** to create and configure a new integration instance.
+1. Search for VirusTotal (API v3).
+1. Click **Add instance** to create and configure a new integration instance.
```

Comment:
please revert the numbering bullets?

Reply:
That's how the numbers in the bullet works in Markdownm